src/js/core/widget/core/Appbar.js

```diff
@@ -413,6 +468,33 @@
 				}
 			};
 
+			/**
+			 * Verification of screen height
+			 * @method _verifyExpanding
+			 * @member ns.widget.core.Appbar
+			 * @protected
+			 */
+			prototype._verifyExpanding = function () {
```

Comment:
The code logic suggests "_setExpanding" name.

Reply:
I wondered about it before. The "_set" is not also good because it is not the "set" we don't have parameter of function ... What you think about "_validate"